### PR TITLE
Moved Confirmation Codes to Getting started for better visibility and added Error Handling section

### PR DESCRIPTION
**Pull request by kunalshah01**

### CODE REVIEW

File: src/api-reference/direct-connects/hotel-service-4/v4.getting-started.markdown [hunk 1]
```diff
@@ -151,3 +151,34 @@ SAP Concur services will send the following HTTP headers in every request. Pleas
...
+
+All HSv4 endpoints use consistent error handling. API responses can have one of the following HTTP response code:
+
+|HTTP Code|Description|
+|---|---|
+|200|Operation successful without any errors.|
+|400|Invalid client request. Request shouldn’t be retried without changing it. Response should follow [Error](./v4.endpoints.md#schemaerror) schema and include OTA Code providing accurate reason for request being invalid.| 
+|401|Request unauthorized. Credentials used for the request are not valid.|
+|500|System error while processing the request. Request can be retried as is at a later time. Response should follow [Error](./v4.endpoints.md#schemaerror) schema. OTA Code is optional for such errors but we expect useful message for better troubleshooting.| 
+
```
Comment (gowthamvelusamy) [inline src/api-reference/direct-connects/hotel-service-4/v4.getting-started.markdown:174]: @kunalshah01  Do you see the value in adding following error codes as well to the above list?
404 -Not Found, 503 - Indicate Outage at the vendor side

File: src/api-reference/direct-connects/hotel-service-4/v4.getting-started.markdown [hunk 2]
```diff
@@ -151,3 +151,34 @@ SAP Concur services will send the following HTTP headers in every request. Pleas
...
 `concur-correlationid`|`string`|-| In order to assist with troubleshooting, SAP Concur provides a unique `correlationId` in the request header. This unique code can be used during troubleshooting as it identifies the API call in the log files.|
 `Accept-Language`|`string`|-| Value of this header will follow language tag of the format LanguageCode-CountryCode, e.g. `en-US`, `fr-CA`, `de-CH` etc.|
+
+## Confirmation Codes <a name="confirmation-codes"></a>
+
+Please note current definitions of `ConfirmationCodeType` differ from what was used in [Hotel Service v2](./hotel-service-2/Introduction.html) (HSv2).
+* `RESERVATION`: Maps to OTA Unique ID Type `14` and has no equivalent for HSv2. This will be the record locator of supplier PNR and will be provided in all subsequent read/modify/cancel request on a booking. This is optional and will be mostly pass through for SAP Concur systems.
+* `SUPPLIER_CONFIRMATION`: **(Required)** Maps to OTA Unique ID Type `40` and represents supplier's confirmation number. This also be provided in all subsequent read/modify/cancel request on a booking, and it is also used in the passive segment created in the GDS. (Equivalent to `14` in HSv2).
```
Comment (gowthamvelusamy) [inline src/api-reference/direct-connects/hotel-service-4/v4.getting-started.markdown:159]: Do we need to communicate about field length constraints for this confirmation code?

**Review by gowthamvelusamy — COMMENTED** [review]
Added couple of review comments.

File: src/api-reference/direct-connects/hotel-service-4/v4.getting-started.markdown (hunk 1 above)
Comment (kunalshah01) [reply src/api-reference/direct-connects/hotel-service-4/v4.getting-started.markdown:174]: No, expectation is to use OTA codes for all validation/application errors and 500 for all system errors (including outage).

File: src/api-reference/direct-connects/hotel-service-4/v4.getting-started.markdown (hunk 2 above)
Comment (kunalshah01) [reply src/api-reference/direct-connects/hotel-service-4/v4.getting-started.markdown:159]: Let's see what constraints we want to add first. Only record locator I believe will be restricted to 6 (but can go up-to) 8 chars.

**Review by gowthamvelusamy — APPROVED** [review]

**Review by RoAnnC1 — APPROVED** [review]
:shipit:

**Pull request merged**